placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesApi.java

```diff
@@ -160,7 +162,8 @@ public PlacesDetailsResponse details(final String placeId) throws IOException {
                 .appendPath(PATH_DETAILS)
                 .appendPath(PATH_JSON)
                 .appendQueryParameter(PARAMETER_KEY, googleApiKey)
-                .appendQueryParameter(PARAMETER_PLACE_ID, placeId);
+                .appendQueryParameter(PARAMETER_PLACE_ID, placeId)
+                .appendQueryParameter(PARAMETER_LANGUAGE, Locale.getDefault().toString());
```

Comment:
hi @sddamico 
Thanks for review. Yeah, it is better to use Locale.getDefault().getLanguage().

Your suggestion to keep in sync language in autocomplete and places details requests sounds logically.
What do you think if I add optional language attribute onto PlacesAutocompleteTextView. If it is populated I will use it, otherwise take device's language as default?